etk/extractors/language_identification_extractor.py

```diff
@@ -33,4 +37,16 @@ def extract(self, text: str, method: str) -> List[Extraction]:
         Returns the empty list of the extractor fails to identify the language in the text.
 
         """
-        pass
+        if method == IdentificationTool.LANGID.name:
+            language = classify(text)[0]
```

Comment:
does this line always return a value or can it be none? If it can be none, this case should be handled and no Extraction object should be returned